airflow/www/views.py

```diff
@@ -837,8 +837,8 @@ def log(self):
         if ti:
             host = ti.hostname
             log_loaded = False
-
-            if socket.getfqdn() == host:
+            
+            if conf.get('webserver', 'FORCE_LOCAL_LOGS') or socket.getfqdn() == host:
```

Comment:
@dantino If we don't hear back from @aoen or @mistercrunch by end of day, let's just fix this to use os.path.exists, and I'll merge it.